zio-http/jvm/src/test/scala/zio/http/ClientServerSentEventSpec.scala

```diff
@@ -0,0 +1,294 @@
+             |""".stripMargin
+        assertEvents(events, Chunk.single(ServerSentEvent(data = "data", eventType = Some("changed-sse"))))
+      },
+      test("Event type can be unset") {
+        // IDEs tend to trim lines on multiline strings, so build event from collection
+        val events = Chunk("event: ", "event:", "event").map { field =>
+          s"""|event: sse
+              |data: data
+              |$field""".stripMargin
+        }.mkString("", "\n\n", "\n\n")
+        assertEvents(events, Chunk.fill(3)(ServerSentEvent(data = "data")))
+      },
+      test("Event type without data is not emitted") {
+        val events =
+          """|event: sse
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.empty)
+      },
+      test("Event id can be overridden") {
+        val events =
+          """|id: 1
+             |data: data
+             |id: 2
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.single(ServerSentEvent(data = "data", id = Some("2"))))
+      },
+      test("Event id can be unset") {
+        // IDEs tend to trim lines on multiline strings, so build event from collection
+        val events = Chunk("id: ", "id:", "id").map { field =>
+          s"""|id: 1
+              |data: data
+              |$field""".stripMargin
+        }.mkString("", "\n\n", "\n\n")
+        assertEvents(events, Chunk.fill(3)(ServerSentEvent(data = "data")))
+      },
+      test("Event id without data is not emitted") {
+        val events =
+          """|id: 1
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.empty)
+      },
+      test("Event id can be a string") {
+        val events =
+          """|id: event-id
+             |data: data
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.single(ServerSentEvent(data = "data", id = Some("event-id"))))
+      },
+      test("Event retry can be overridden") {
+        val events =
+          """|retry: 1
+             |data: data
+             |retry: 2
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.single(ServerSentEvent(data = "data", retry = Some(2.milliseconds))))
+      },
+      test("Event retry can be unset") {
+        // IDEs tend to trim lines on multiline strings, so build event from collection
+        val events = Chunk("retry: ", "retry:", "retry").map { field =>
+          s"""|retry: 1
+              |data: data
+              |$field""".stripMargin
+        }.mkString("", "\n\n", "\n\n")
+        assertEvents(events, Chunk.fill(3)(ServerSentEvent(data = "data")))
+      },
+      test("Event retry without data is emitted") {
+        val events =
+          """|retry: 1
+             |
+             |""".stripMargin
+        assertEvents(events, Chunk.single(ServerSentEvent(data = "", retry = Some(1.millisecond))))
+      },
+      test("Event retry that is not an integer is not emitted") {
```

Comment:
Postman will emit events with provided `retry` value. Browser ignores them. Specification says to ignore.